src/main/resources/application-worker.yaml

```diff
@@ -30,3 +34,8 @@ rhsm-subscriptions:
     internal:
       url: http://${CLOUDIGRADE_INTERNAL_HOST}:${CLOUDIGRADE_INTERNAL_PORT}/internal/api/cloudigrade/v1
       presharedKey: ${CLOUDIGRADE_PSK:}
+  tally-summary-producer:
+    back-off-initial-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_INITIAL_INTERVAL}
+    back-off-max-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MAX_INTERVAL}
+    back-off-multiplier: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MULTIPLIER}
+    max-attempts: ${TALLY_SUMMARY_PRODUCER_MAX_ATTEMPTS}
```

Comment:
I'd prefer to remove the definitions up on lines 5-8 and instead define everything like this:

```
tally-summary-producer:
    back-off-initial-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_INITIAL_INTERVAL:1s}
    back-off-max-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MAX_INTERVAL:1m}
    back-off-multiplier: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_MULTIPLIER:2}
    max-attempts: ${TALLY_SUMMARY_PRODUCER_MAX_ATTEMPTS:5}
```

The big reason to define an property up at the top that's meant to have the same name as an environment variable is for when we would otherwise have to nest expressions (e.g. we can't nest expressions like `${MY_VAR:${clowder.jsonPathHere}}` or `${CLOUDIGRADE_INTERNAL_HOST:$CLOUDIGRADE_HOST}`).